packages/cli/src/bundle/overrides.ts

```diff
@@ -13,29 +13,34 @@ export type BundleConfigOverrides = {
   assetsDest?: string;
   treeShake?: boolean;
   unstableTransformProfile?: TransformProfile;
+  indexedRamBundle?: boolean;
 };
 
+export const overridableBundleOptions: readonly (keyof BundleConfigOverrides)[] =
```

Comment:
```suggestion
export const overridableCommonBundleOptions: readonly (keyof BundleConfigOverrides)[] =
```

This lines it up with the naming you established with `CLICommonBundleOptions`.